Makefile

```diff
@@ -178,7 +179,7 @@ clean: tidy clean-tools
 	@$(MAKE) clean -C multi_boot/collect_rings
 
 	$(RM) $(SAMPLE_SUBDIR)/*.bin $(MID_SUBDIR)/*.s
-	find . \( -iname '*.1bpp' -o -iname '*.4bpp' -o -iname '*.8bpp' -o -iname '*.gbapal' -o -iname '*.lz' -o -iname '*.rl' -o -iname '*.latfont' -o -iname '*.hwjpnfont' -o -iname '*.fwjpnfont' \) -exec $(RM) {} +
+	find . \( -iwholename './data/maps/**/interactables.bin' -iwholename './data/maps/**/items.bin' -iwholename './data/maps/**/enemies.bin' -iwholename './data/maps/**/rings.bin' -o -iname '*.1bpp' -o -iname '*.4bpp' -o -iname '*.8bpp' -o -iname '*.gbapal' -o -iname '*.lz' -o -iname '*.rl' -o -iname '*.latfont' -o -iname '*.hwjpnfont' -o -iname '*.fwjpnfont' \) -exec $(RM) {} +
```

Comment:
Why not just `*.bin`?

Reply:
I think the *.bin was only removed from here because of these specific bins which you have now removed. We can filter out the specific bin from here if we ever introduce other

Reply:
`./data/recordings` still contains .bin files, so we still gotta make it folder-specific at least.